packages/stockflux-watchlist/src/components/watchlist-card/WatchlistCard.js

```diff
@@ -179,7 +194,11 @@ const WatchlistCard = ({
           </div>
           <div
             className="card-bottom"
-            onClick={() => bindings.onDropOutside(symbol, stockData.name)}
+            onClick={() => {
+              setDragOutcome('');
+              bindings.onDropOutside(symbol, stockData.name);
+            }}
+            onMouseDown={() => setDragOutcome('CHART')}
```

Comment:
Asking for clarification, the `setDragOutcome` is done twice to set it back to the default value?

Reply:
It should not be done twice, removed this instance of it as it will be dealt with `setDragOutside`.